### ISSUE TITLE
Support python 3

### ISSUE THREAD

**Issue by bmcfee** [marl/medleydb#30]
```python
In [1]: import medleydb
  File "/home/bmcfee/data/medleydb/medleydb/__init__.py", line 20
    the top level Audio folder for MedeleyDB."""
                                                
^
SyntaxError: Missing parentheses in call to 'print'
```

.. or at least do a ``from __future__ import print_function``.  But I suspect there are other less obvious gotchas kicking around.

(also `Medley` is misspelled :))

**Comment by bmcfee** [comment]
note: i took a stab at automating this via [2to3](https://docs.python.org/2/library/2to3.html), but it didn't work immediately so i gave up.

**Comment by rabitt** [comment]
closed in 410abe3
